assets/app/views/deployments.html

```diff
@@ -15,13 +15,24 @@
             <th>Trigger</th>
           </tr>
         </thead>
-        <tbody ng-if="(deploymentConfigs | hashSize) === 0 && ((deployments | hashSize) === (deploymentsByDeploymentConfig[''] | hashSize))">
+        <!-- message doesnt show right when there are both dcs and rcs and they are all filtered -->
+        <tbody ng-if="(deploymentConfigs | hashSize) === 0
+                      && 
+                      (
+                        ((deploymentsByDeploymentConfig | hashSize) - (unfilteredDeploymentConfigs | hashSize) == 0)
+                        ||
+                        (
+                          ((deploymentsByDeploymentConfig | hashSize) - (unfilteredDeploymentConfigs | hashSize) == 1)
+                          &&
+                          deploymentsByDeploymentConfig['']
+                        )
+                      )">
```

Comment:
This is a lot of logic for an `ng-if`. Can we move it into the controller? I'm not clear on what this is doing.